prowler/lib/outputs/jira/jira.py

```diff
@@ -209,6 +230,31 @@ def get_timestamp_from_seconds(seconds: int) -> datetime:
         """
         return (datetime.now() + timedelta(seconds=seconds)).isoformat()
 
+    def get_basic_auth(self) -> None:
+        """Get the access token using the mail and api token
+
+        Returns:
+            - None
+
+        Raises:
+            - JiraBasicAuthError: Failed to authenticate using basic auth
+        """
+        try:
+            user_string = f"{self._client_mail}:{self._api_token}"
+            self._access_token = base64.b64encode(user_string.encode("utf-8")).decode(
+                "utf-8"
+            )
```

Comment:
Why do you need to decode?

Reply:
It's done this way because I need an string at the end. If I remove the decode part i'll get only the bytes